src/blacksmith/sd/_async/adapters/nomad.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
you have  to create a file in the docs

to refer all your new file.

docs/source/develop/sd/nomad.rst

```
Nomad
======

.. automodule:: blacksmith.sd._async.adapters.nomad
   :members:

```


and refer it in the index.